config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+wRoirU8Y5anmaW+6XaPa0lkPjDiuhZPOqlKL2oCRY35b32nohogV4yukhkwRr1tp3QDaLzfT2HBtZ2McixqBv6sQ+HIBoBgF7/sx/iO1VmUzoXFAKnfk7vN5psYh54ZB+q5BFs6SmrObE31mVIKU93xUnlARUwB9QtGd0xkmQCLV2YYnRb1uzdtCiYZDJzr9tsRPgKCbGviuQrYFWCvCpdHitx/uZ5PN1Km3jJNq0RrhB/HdUKpAtcijdbW61YflfPiPgpmd1I1vFs7DH3Xp3OGZRppxQGzja6+LcSQYNaijO9JsEVmJsjA6hg+4Zf10N4LqQl5cXvNXKx9KAqfE/KrLDTJp/Oxb6Tqk4Cl32u26FO/9iMi9gaEp8aMxALD2NpD949UG3CIz5tnV5nkWFqoFoskPwzfGmPJU--/QvvEh0H87UJRT+z--qOVcsDuWhDo2dUpKr1G0eA==
```

Comment:
add this file in .gitignore , credentials , keys etc. should all be encrypted and not pushed to repo